xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/UsersClass.java

```diff
@@ -210,16 +212,6 @@ public BaseProperty newPropertyfromXML(Element ppcel)
         return selectlist;
     }
 
-    @Override
-    public void fromList(BaseProperty<?> property, List<String> list)
```

Comment:
I must be missing something, but don't you want to filter empty values here ?

Reply:
So in the context of the bug I fixed the call line 169: https://github.com/xwiki/xwiki-platform/pull/1800/files#diff-9db5305f72027b429d4ade2c90c36c1b2b49151d72b07281092732a02f01bc4eR169

Reply:
I don't understand, `fromList(BaseProperty<?> property, List<String> list)` does not seem to be reusing  `fromStringArray(String[] strings)`.

Reply:
That's the contrary: fromStringArray should have used fromList, as in GroupsClass#fromStringArray. So I performed same change in both classes.

Reply:
My point here is that if someone calls `UsersClass#fromList(myproperty, mylist)`, the empty element from `mylist` won't be filtered because that's the default behavior of `ListClass#fromList(BaseProperty<?> property, List<String> list)` from what I understand.

Reply:
> My point here is that if someone calls UsersClass#fromList(myproperty, mylist), the empty elements from mylist won't be filtered because that's the default behavior of ListClass#fromList(BaseProperty<?> property, List<String> list) from what I understand.

Yes that's right, I kept same behaviour as before for this call. I'm fixing only the call from #fromStringArray.

Reply:
But that's wrong. We want to filter empty values in general in GroupClass and UserClass since they don't make sense, it does not make sense to do it only for String arrays and not String lists.

Reply:
ok I'll put back the override with a filtering of empty values by default then.